core/embed/lib/fonts/fonts_types.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _FONTS_TYPES_H
```

Comment:
any particular reason you made this a separate file, instead of putting the data into fonts.h ?

Reply:
I put these fefinitions to `fonts.h` on rebase.